src/Dapr.Actors/Runtime/ActorTimer.cs

```diff
@@ -18,49 +18,38 @@ internal class ActorTimer : IActorTimer
         public ActorTimer(
             Actor owner,
             string timerName,
-            Func<object, Task> asyncCallback,
-            object state,
-            TimeSpan dueTime,
-            TimeSpan period)
+            TimerInfo timerInfo)
         {
             this.owner = owner;
             this.Name = timerName;
-            this.AsyncCallback = asyncCallback;
-            this.State = state;
-            this.Period = period;
-            this.DueTime = dueTime;
+            this.TimerInfo = timerInfo;
         }
 
         public string Name { get; }
 
-        public TimeSpan DueTime { get; }
+        internal TimerInfo TimerInfo { get; }
```

Comment:
Any good reason why this is internal visibility? The type is already internal, and now it has a mix of public and internal properties